### PR TITLE
chore(docs): Getting Started documentation

### PR DESCRIPTION
**Pull request by GreggSetzer**
# Objective
Add more details to the Getting Started documentation page.

## Overview
This PR introduces additional details to the Getting Started documentation. For consistency, please let me know if you prefer one way or the other.

I've reviewed all references to the project name `projen`. I'm following this pattern:
1. If the project title is referenced as the first word in a sentence, I am capitalizing the project name (`Projen`).
2. If used in the middle of a sentence or code comments, I'm using the lowercase version (`projen`).

As part of the updates to this page, I am introducing a number of code samples using TypeScript for demonstration purposes. The goal is to provide a more comprehensive demo of how we can personalize our projects.

---
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by GreggSetzer** [comment]
@mbonig

File: docs/introduction/getting-started.md [hunk 1]
```diff
@@ -4,91 +4,324 @@ sidebar_position: 1
...
+- For a Python project, the file will be `.projenrc.py`.
+- If you choose to use JSON, the file will be `.projenrc.json`.
 
-```shell
-$ npx projen --watch
+Typically, the projen RC file will be in the same language as the project.
+
+## Adding Dependencies
+
+You can add dependencies using projen's API. For example, to add a runtime dependency:
```
Comment (mbonig) [inline docs/introduction/getting-started.md:79]: This should also include adding dependencies by setting the property on the constructor.

File: docs/introduction/getting-started.md [hunk 2]
```diff
@@ -4,91 +4,324 @@ sidebar_position: 1
 
 # Getting Started
 
-projen doesn't need to be installed. You will be using [npx](https://docs.npmjs.com/cli/v7/commands/npx) to run _projen_ which takes care of all required setup steps.
+Welcome to projen! This guide will walk you through the essential steps to get up and running
```
Comment (mrgrain) [inline docs/introduction/getting-started.md:7]:
```suggestion
This guide will walk you through the essential steps to get up and running 
```

File: docs/introduction/getting-started.md [hunk 3]
```diff
@@ -4,91 +4,324 @@ sidebar_position: 1
...
-projen doesn't need to be installed. You will be using [npx](https://docs.npmjs.com/cli/v7/commands/npx) to run _projen_ which takes care of all required setup steps.
+Welcome to projen! This guide will walk you through the essential steps to get up and running 
+with a TypeScript project, covering much of what you'll typically use. By the end of this guide, you'll have 
+a solid foundation to build and manage your projects efficiently with projen.
 
-:::tip
-To make things easier from here on out, we recommend you create an alias for projen:
+> Note: While this guide uses TypeScript as an example, projen supports a variety of project types including 
+> JavaScript, Python, Java, and more. The concepts and steps outlined here can be adapted to other languages 
+> and project types as well.
```
Comment (mrgrain) [inline docs/introduction/getting-started.md:13]: We need to use the Docusaurus [Admonitions](https://docusaurus.io/docs/markdown-features/admonitions)
```suggestion
:::info
While this guide uses TypeScript as an example, projen supports a variety of project types including 
JavaScript, Python, Java, and more. The concepts and steps outlined here can be adapted to other languages 
and project types as well.
:::
```

File: docs/introduction/getting-started.md [hunk 4]
```diff
@@ -4,91 +4,324 @@ sidebar_position: 1
...
-...
+```bash
+mkdir demo
+cd demo
+npx projen new typescript
 ```
 
-## Modifying projen
+Replace `typescript` with the type of project you want to create, such as `node` for a Node.js project or `python` for 
+a Python project.
```
Comment (mrgrain) [inline docs/introduction/getting-started.md:45]:
```suggestion
a Python project. You can get a complete list of available project types by running `npx projen new --help`.
```

File: docs/introduction/getting-started.md [hunk 5]
```diff
@@ -4,91 +4,324 @@ sidebar_position: 1
...
+
+### Git Ignore Settings:
+
+Customize the `.gitignore` file to include or exclude specific files and directories.
+
+```ts
+project.gitignore.addPatterns('dist/', 'node_modules/');
+```
+
+### Pull Request Template:
```
Comment (mrgrain) [inline docs/introduction/getting-started.md:262]: I reckon this should go into `GitHub Integration`  https://projen.io/docs/integrations/github/

**Review by mrgrain — APPROVED** [review]

**Pull request merged**